components/MintPage/MintV2Page.tsx

```diff
@@ -0,0 +1,168 @@
+import { useRef, useState } from "react"
+import { Button } from "../../shared/Button"
+import Media from "../../shared/Media"
+import Layout from "../Layout"
+import MintCTAButton from "./MintCTAButton"
+import useShakeEffect from "../../hooks/useShakeEffect"
+import MintingModal from "./MintV2/MintingModal"
+import SuccessModal from "./MintV2/SuccessModal"
+import useCre8orMintV2 from "../../hooks/mintDay/useCre8orMintV2"
+
+const MintV2Page = () => {
+  const [mintQuantity, setMintQuantity] = useState(0)
+  const minusRef = useRef()
+  const mintRef = useRef()
+  const [isMintLoading, setIsMintLoading] = useState(false)
+  const [openSuccessModal, setOpenSuccessModal] = useState(false)
+  const { mint } = useCre8orMintV2()
+
+  const increateAmount = () => {
+    setMintQuantity(mintQuantity + 1)
+  }
+
+  const decreaseAmount = () => {
+    if (mintQuantity === 0) return
+    setMintQuantity(mintQuantity - 1)
+  }
+
+  const mintNFT = async () => {
+    if (!mintQuantity) return
+
+    setIsMintLoading(true)
+    const response = await mint(mintQuantity)
+    if (!response.err) {
+      setIsMintLoading(false)
+      setOpenSuccessModal(true)
+      return
```

Comment:
why do we need a return here? we could remove `setIsMintLoading(false)` and the return correct?

Reply:
removed duplicated code 🚮